src/rules/function-url-quotes/__tests__/index.js

```diff
@@ -145,6 +145,10 @@ testRule(rule, {
     code: "@font-face { font-family: 'foo'; src: url($variable + \"foo.ttf\" + $variable); }",
   }, {
     code: "a { background: url(\"/images/my_image@2x.png\") }",
+  }, {
+    code: "@-moz-document url-prefix() {}",
+  }, {
+    code: "a { background: url() }",
```

Comment:
@Nitive Can you update this PR to [add](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#adding-options-to-existing-rules) an `except: ["empty"]` optional secondary option, rather than change the *default* behaviour of the rule?

You'll find a similar implementation at:

- https://github.com/stylelint/stylelint/blob/master/src/rules/rule-non-nested-empty-line-before/index.js#L32
- https://github.com/stylelint/stylelint/blob/master/src/rules/rule-non-nested-empty-line-before/index.js#L65-L69
- https://github.com/stylelint/stylelint/blob/master/src/rules/rule-non-nested-empty-line-before/README.md#optional-secondary-options-1
- https://github.com/stylelint/stylelint/blob/master/src/rules/rule-non-nested-empty-line-before/__tests__/index.js#L133-L151